sale_order_invoice_date/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+Display the invoice date on Sales Order analysis reports.
```

Comment:
@ivantodorovich Could you just add a line that indicates user that invoice date is the last one if several invoices for the same sale order ?

Reply:
Done 👌🏻 thanks!!